docs/index.md

```diff
@@ -2,5 +2,52 @@
 title: "Introduction"
 weight: 10
 ---
+The AsyncAPI Generator is a tool that generates anything you want using the **[AsyncAPI file](asyncapi-file.md)** and **[template](template.md)** that are supplied as inputs to the AsyncAPI Generator CLI. The Generator was built with extensibility in mind; you can use the generator to generate anything you want provided that it can be defined in a template, such as code, diagrams, markdown files, microservices and applications for example. A number of [community-maintained templates](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate) are now available for immediate usage.
 
->This document is under construction.
+### Use Cases of the Generator
+- Generation of interactive and understandable API documentation.
```

Comment:
```suggestion
- Generation of interactive and understandable API documentation
```

Consistency is key... since we don't use punctuation in the rest of the bullet points then let's remove this single period ✌🏽